### PR TITLE
feat: Add PriceAreaCode to grid areas.

### PR DESCRIPTION
**Pull request by defectiveAi**

### CODE REVIEW

File: libs/dh/market-participant/feature-gridarea/src/lib/overview/dh-market-participant-gridarea-overview.component.html [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
 <ng-container *transloco="let t; read: 'marketParticipant.gridAreas'">
   <!-- drawer -->
-  <watt-drawer #drawer (closed)="drawerClosed()" [size]="'large'">
+  <watt-drawer #drawer (closed)="drawerClosed()" [size]="'normal'">
```
Comment (ManBearTM) [inline libs/dh/market-participant/feature-gridarea/src/lib/overview/dh-market-participant-gridarea-overview.component.html:20]:
```suggestion
  <watt-drawer #drawer (closed)="drawerClosed()" size="normal">
```

File: libs/dh/market-participant/feature-gridarea/src/lib/details-header/dh-market-participant-gridarea-details-header.component.html [hunk 2]
```diff
@@ -18,21 +18,52 @@
...
+            },
+            {
+              header: t('lossMeteringPoint'),
+              value: undefined | dhEmDashFallback
+            },
+            {
+              header: t('correctionMeteringPoint'),
+              value: undefined | dhEmDashFallback
+            }
+          ]
```
Comment (ManBearTM) [inline libs/dh/market-participant/feature-gridarea/src/lib/details-header/dh-market-participant-gridarea-details-header.component.html:59]: question: is this "final" or will the static array definition be replaced with something more dynamic?

Comment (defectiveAi) [reply libs/dh/market-participant/feature-gridarea/src/lib/details-header/dh-market-participant-gridarea-details-header.component.html:59]: More entries will be added in future PRs, but the array will continue be static.

Comment (defectiveAi) [reply libs/dh/market-participant/feature-gridarea/src/lib/details-header/dh-market-participant-gridarea-details-header.component.html:59]: @ManBearTM

Comment (dzhavat) [reply libs/dh/market-participant/feature-gridarea/src/lib/details-header/dh-market-participant-gridarea-details-header.component.html:59]: @defectiveAi Bjørn has today and tomorrow off. I can help with the PR review instead.

> More entries will be added in future PRs, but the array will continue to be static.

In this case let's remove the array (and the loop) and have each section as a separate HTML elements. There's going to be some repetition, but we'll avoid the above logic in the template.

Comment (defectiveAi) [reply libs/dh/market-participant/feature-gridarea/src/lib/details-header/dh-market-participant-gridarea-details-header.component.html]: Fixed

**Review by dzhavat — APPROVED** [review]

**Pull request merged**